packages/components/src/components/text/text.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React, { HTMLProps } from 'react';
+import { isEmptyObject } from '@deriv/shared';
+import { TGenericStringValueObject } from '../types';
+
+interface ITextProps extends HTMLProps<HTMLElement> {
+    align?: string;
+    weight?: string;
+    line_height?: string;
+    styles?: TGenericStringValueObject;
+}
+type TTextProps = Omit<ITextProps, 'size'> & {
+    size: string;
+};
```

Comment:
Why do we need to Omit `size`?

Reply:
HTMLProps<HTMLElement> has a size which is a number, and as we want it to be a string, we have to omit it & set our own size. so my suggestion to write it all in 1 type is 🙂:
```suggestion
type TTextProps = Omit<HTMLProps<HTMLElement>, 'size'> & {
    align?: string;
    weight?: string;
    line_height?: string;
    styles?: React.CSSProperties;
    size: string;
};
```

Reply:
@jim-deriv better to write into one type as per @maryia-binary's suggestion